qpc/cred/tests_add.py

```diff
@@ -151,7 +151,7 @@ def test_add_host_cred(self):
                              type=NETWORK_CRED_TYPE,
                              filename=TMP_KEY,
                              password=None, sudo_password=None,
-                             ssh_passphrase=None)
+                             ssh_passphrase=None, become_method='sudo')
```

Comment:
should have test where become_method is `None` as well; expecting that the api will add the default.

Reply:
I do a test for this inside the quipucords/api/credential/tests_credential.py (test_hostcred_default_become_method). I can put one here as well - but it will only be able to test that the credential is successfully created when become_method is None (if it follows suit with the other tests).